Makefile

```diff
@@ -240,5 +240,5 @@ yamllint: $(YAMLLINT_BIN)
 
 .PHONY: _protoc ## Install all required tools for use with this Makefile.
 _protoc:
-	$(GO) install $(PROTOC_GEN_GO)
-	$(GO) install $(PROTOC_GEN_GO_GRPC)
\ No newline at end of file
+	GOBIN=$${PWD}/bin $(GO) install $(PROTOC_GEN_GO)
```

Comment:
I don't think `make generate-proto` will find these binaries unless someone adds `$PWD/bin` so their path.

I'd be keen to stick them under `out/bin` or `out/tools` also. I intend on moving the binaries we build to `out/bin` at some point so we have easy clean up.

Reply:
the makefile does add `$PWD/bin` to the PATH. https://github.com/tinkerbell/tink/blob/eef654590132fbd291449904d3385100e386120f/Makefile#L4